### ISSUE TITLE
[circleci] build broken due to circleci/node:8 changes

### ISSUE THREAD

**Issue by stefan-guggisberg** [adobe/helix-cli#353]
**Description**
as of dec 28 circleci build jobs (i.e. `npm test`) fail with:

```
/home/circleci/repo/node_modules/nodegit/dist/nodegit.js:16
    throw ex;
    ^

Error: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
```
node version hasn't changed (v8.13).

**To Reproduce**
- Got to CircleCi dashboard
- open a failing `helix-cli` `build` job
- open `Spin up  Environment` step and check `sha256` of `circleci/node:8` image
- do the same with last successful `build` job  

**Expected behavior**
build job shouldn't fail if it succeeded previously and there were no changes to the code or circleci config.

**Comment by kptdobe** [comment]
I applied the same tmp fix to the smoke test instance: https://github.com/adobe/helix-continuous/commit/b1c4450c61fc786e9f0c6a3a5d04209d2f47777c

**Comment by stefan-guggisberg** [comment]
problem analysis:

`circleci/node:8` used to resolve to `circleci/node:8.12-jesssie` (jessie -> debian 8)

since a couple of hours ago `circleci/node:8` resolves to `circleci/node:8.13-stretch` (stretch -> debian 9)

on debian 9 the following error occurs when running the tests:

```
Error: libcrypto.so.1.0.0: cannot open shared object file: No such file or directory
```

solution: use latest node 8 version on debian 8, i.e. `circleci/node:8-jessie`

**Comment by stefan-guggisberg** [comment]
fixed in 25ec7cc

**Comment by kptdobe** [comment]
I patched the CI too: https://github.com/adobe/helix-continuous/commit/227ddd8c3de682dd64f5b60c0b4a0c12256e2f76
